### PR TITLE
381524 Declare variables for Cosmos database & container

### PR DESCRIPTION
**Pull request by rtasalem**
https://dev.azure.com/defragovuk/DEFRA-FFC/_workitems/edit/381524

### CODE REVIEW

**Comment by rtasalem** [comment]
Purpose of this PR was to declare the name of the Cosmos database (`ffc-sfd-customer-receiver-queries`) and container (`queries-container`) in the config so the `COSMOS_QUERIES_DATABASE` and `COSMOS_QUERIES_CONTAINER` variables can be removed from `.env` and `docker-compose.yaml`. Rebuilt container and tested - can confirm working as expected i.e. generic message sent through receiver is saving to Cosmos.

File: app/queries/send-notification.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+const { v4: uuidv4 } = require('uuid')
+const NotifyClient = require('notifications-node-client').NotifyClient
+
+const sendNotification = async (message) => {
+  const notifyClient = new NotifyClient(process.env.NOTIFY_API_KEY)
+
+  await notifyClient.sendEmail(
+    process.env.NOTIFY_TEMPLATE_ID,
+    'rana.salem@defra.gov.uk',
```
Comment (simonnineteen90) [inline app/queries/send-notification.js]: Could change this to be an ENV variable instead so you dont get spammed when others use the service

**Review by simonnineteen90 — COMMENTED** [review]
Same as previous PR i think we can use the appConfig.yaml for any values that are the same across every environment to avoid duplication

**Comment by rtasalem** [comment]
> Same as previous PR i think we can use the appConfig.yaml for any values that are the same across every environment to avoid duplication

@simonnineteen90 Done, have removed the duplicates and put them in SND3 instead and service is deploying with no errors.

File: app/queries/send-notification.js (hunk 1 above)
Comment (rtasalem) [reply app/queries/send-notification.js]: Done, new env var is `NOTIFY_TEST_EMAIL` (see `docker-compose.yaml`).

**Comment by rtasalem** [comment]
Deployment for this service is now successful. To summarise, like the messages receiver, we're only using SND3 for the time being. No errors in the pod logs (see below):

image

This is now ready for another review.

**Review by simonnineteen90 — APPROVED** [review]

**Pull request merged**